templates/epub2/toc.xhtml.ejs

```diff
@@ -9,13 +9,16 @@
 <body>
 <h1 class="h1"><%= tocTitle %></h1>
 <% content.forEach(function(content, index){ %>
-<p class="table-of-content">
-    <a href="<%= content.href %>"><%= (1+index) + ". " + (content.title || "Chapter "+ (1+index)) %>
-        <% if(content.author.length){ %>
-        - <small class="toc-author"><%= content.author.join(",") %></small>
-        <% } %>
-        <% if(content.url){ %><span class="toc-link"><%= content.url %></span><% }else{ %><span class="toc-link"></span><% } %>
-    </a>
-</p><% }) %>
+    <% if(!content.excludeFromToc){ %>
```

Comment:
@cyrilis Here, with current 'API', if the author doesn't take the care to have all `beforeToc` contents first in the array, having instead mixed `true`/`false` contents along the array, the order of this EPUB 2 'meta' TOC will be different from the one on the spine and the book. Just another issue I found by having the property `beforeToc` more permissive.